permafrost/templates/permafrost/permafrostrole_form.html

```diff
@@ -132,16 +132,17 @@ <h6 class="m-0 text-primary font-weight-bold">
                     <div class="col-6">
                         {% for permission in category.optional %}
                         <div class="d-flex align-items-center">
-                            <span>{{ permission.name }}</span>
-                            
+                            <label for="permission">{{ permission.name }}</label>
+
                             {% if permission.short_description %}
                             <i class="ml-2 fas fa-info-circle text-gray-600" aria-label="{% trans 'Info' %}"></i>
                             {% endif %}
-                            <input 
-                                class="ml-auto" 
-                                type="checkbox" 
-                                name="{{ form.permissions.name }}" 
+                            <input
+                                class="ml-auto"
+                                type="checkbox"
+                                name="{{ form.permissions.name }}"
                                 value="{{ permission.id }}"
+                                id="permission"
```

Comment:
this is part of a for loop- they need different ids. `permission.id` should work i think

Reply:
👍

Reply:
don't forget to update the "for" in the label as well :)